### ISSUE TITLE
Please support prefers-reduced-motion for skeleton states

### ISSUE THREAD

**Issue by carmacleod** [IBM/carbon-components#2217]
Some people feel sick when they experience unexpected motion on the web.
The skeleton states animations may be a problem for these people.
There's a relatively new [media query called prefers-reduced-motion](https://caniuse.com/#search=prefers-reduced-motion) that was introduced by Apple in Safari, and is supported in Firefox, and soon to be supported in Chrome.
Please consider supporting this so that if a user chooses the "Reduce Motion" setting in their browser/OS, the skeleton states remain static and they do not animate.

- [CSS Tricks Article: _"An Introduction to the Reduced Motion Media Query"_](https://css-tricks.com/introduction-reduced-motion-media-query/)
- [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
  - including an example that is supported in Safari, Firefox, and the [upcoming Chrome 74](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)

**Comment by joshblack** [comment]
Agreed @carmacleod! Thanks for bringing this up! We should support this up in Carbon elements too so that any motion guidance for IDL comes with this built-in, too 👍

**Comment by joshblack** [comment]
cc @shixiedesign
